src/test/java/com/ss/ita/rozetka/ui/SideModalMenuTest.java

```diff
@@ -0,0 +1,55 @@
+package com.ss.ita.rozetka.ui;
+
+import com.ss.ita.rozetka.ui.elements.Header;
+import com.ss.ita.rozetka.ui.modals.HamburgerModal;
+import com.ss.ita.rozetka.ui.pages.HomePage;
+import com.ss.ita.rozetka.ui.testUtils.TestRunner;
+import io.qameta.allure.Issue;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SideModalMenuTest extends TestRunner {
+
+    @Test
+    @Issue("LVTAQC630-28")
+    public void verifyChangingTown() {
+        String town1 = "Одеса";
+        String town2 = "Львів";
+
+        Header header = new HomePage()
+                .open()
+                .getHeader();
+        HamburgerModal hamburgerModal = header.openHamburgerModal();
```

Comment:
rename hamburger modal

Reply:
I waiting for it in another PR during 5 days...